documentation/common/attributes.adoc

```diff
@@ -80,6 +80,9 @@
 
 :CronExpression: link:http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/tutorial-lesson-06.html[cron expression^]
 
+:OperatorHub: link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/[OperatorHub.io^]
```

Comment:
The link attribute should go to https://operatorhub.io/

Reply:
I'm stupid :-(

Reply:
It's easily done when copying and pasting attributes.